docs/integration/categories/applicative/delinea_pra.md

```diff
@@ -0,0 +1,38 @@
+uuid: 049b3dfd-8f67-40b6-a22d-b86ba3b42a05
+name: Delinea PRA
+type: intake
+
+## Overview
+
+Delinea’s Privileged Remote Access (PRA) is a secure session‐broker solution that lets organizations grant, monitor and audit elevated RDP, SSH and SFTP connections to critical systems without exposing underlying credentials to end users. PRA integrates with your vault, identity providers and SIEMs to enforce just-in-time, least-privilege access policies, record every keystroke and file transfer, and stream real-time audit events for compliance and threat detection. By isolating sessions, injecting credentials on the fly and capturing detailed forensic logs, Delinea PRA reduces risk from shared accounts, lateral movement and credential theft while delivering complete visibility into who accessed what, when and how.
+
+## Configure
+
+This setup guide will show you how to forward your Delinea logs to Sekoia.io by means of a syslog transport channel.
+
+### Delinea PRA configuration
+
+#### Configuring a Secure TCP Syslog or CEF External Audit Server in Secret Server
+
+1. Navigate to `Admin` > `Application`. The Application page appears.
+2. Click the Edit button at the top right of the page.
+3. Click to select the `Enable Syslog/CEF Log Output` check box. A syslog/CEF section expands below.
+4. Type IP address or name for the IIS server hosting the syslog/CEF server in the `Syslog/CEF Server` text box.
+5. Type the port number where the logging information will be passed (6514 is the default port for secure TCP syslog) in the `Syslog/CEF Port` text box.
+6. Click the `Syslog/CEF Protocol` dropdown list and select `Secure TCP`. Secure TCP means either TLS v1.2 or v1.1 because other versions of SSL, such as SSL v3 and TLS v1.0, have known weaknesses.
+7. Click to select `Syslog/CEF Time Zone` list box to `UTC Time` or `Server Time`, depending on your preference.
```

Comment:
```suggestion
7. Click to select `Syslog/CEF Time Zone` list box to `UTC Time`
```
I strongly suggest UTC only. The date in the events must satisfy [this criteria](https://docs.sekoia.io/xdr/FAQ/datetime/#timezone).